src/enums/ClaimConditionType.ts

```diff
@@ -0,0 +1,27 @@
+enum ClaimConditionType {
+  StartTime,
+  LimitPerWallet,
+  MerkleRoot,
+  WaitTimeBetweenClaims,
+}
+
+/**
+ * Questions:
+ *
+ * 1. Can there only be a single `startTimestamp` condition? What is a `startTimestamp` condition?
+ *
+ * 2. What is `maxMintSupply`?
```

Comment:
maximum number of mintable nft per condition

Reply:
must be more than 0.

Reply:
> maximum number of mintable nft per condition

What does "per condition" mean in this context?

Reply:
mint conditions. the array of mint conditions. each individual of the conditions can have different max mint supply